src/dev/jest/config.js

```diff
@@ -82,8 +77,7 @@ export default {
     'integration_tests/'
   ],
   transform: {
-    '^.+\\.js$': '<rootDir>/src/dev/jest/babel_transform.js',
-    '^.+\\.tsx?$': '<rootDir>/src/dev/jest/ts_transform.js',
+    '^.+\\.(jsx?|tsx?)$': '<rootDir>/src/dev/jest/babel_transform.js',
```

Comment:
There are a lot of jsx mentions, we don't need to worry about that

Reply:
👍